packages/@lwc/state/src/index.ts

```diff
@@ -130,12 +131,7 @@ export const defineState: DefineState = <
   Args extends unknown[],
   ContextShape,
 >(
-  defineStateCallback: (
-    atom: MakeAtom,
-    computed: MakeComputed,
-    update: MakeUpdate,
-    fromContext: MakeContextHook<ContextShape>,
-  ) => (...args: Args) => InnerStateShape,
+  defineStateCallback: (api: ExposedAPIs<ContextShape>) => (...args: Args) => InnerStateShape,
```

Comment:
Does all the type inference continue to work with this change? I'm in favor so long as the types (internal and external) can still be inferred when writing & consuming a state manager.

Reply:
yep, the type inference is working!